docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
         - PYTHON_VER=latest
     volumes:
       - ./.docker/wait-for-it.sh:/bin/wait-for-it.sh
-    command: ["/bin/wait-for-it.sh", "postgres_db:5432", "-s", "-t", "0", "--", "python3", "runtests.py"]
+    command: ["/bin/bash", "/bin/wait-for-it.sh", "postgres_db:5432", "-s", "-t", "0", "--", "python3", "runtests.py"]
```

Comment:
To fix:
 
![image](https://user-images.githubusercontent.com/2521996/136254923-77eaca83-93e9-4daa-99c8-a9a104f6098a.png)